test/Microsoft.Azure.SignalR.AspNet.Tests/Infrastructure/VerifiableLoggedTest.cs

```diff
@@ -14,18 +14,18 @@ public VerifiableLoggedTest(ITestOutputHelper output) : base(output)
         {
         }
 
-        public virtual IDisposable StartVerifiableLog(out ILoggerFactory loggerFactory, [CallerMemberName] string testName = null, Func<WriteContext, bool> expectedErrorsFilter = null)
+        public virtual IDisposable StartVerifiableLog(out ILoggerFactory loggerFactory, [CallerMemberName] string testName = null,Func<WriteContext, bool> expectedErrorsMatch = null)
         {
             var disposable = StartLog(out loggerFactory, testName);
 
-            return new VerifyNoErrorsScope(loggerFactory, disposable, expectedErrorsFilter);
+            return new VerifyNoErrorsScope(loggerFactory, disposable, expectedErrorsMatch);
         }
 
-        public virtual IDisposable StartVerifiableLog(out ILoggerFactory loggerFactory, LogLevel minLogLevel, [CallerMemberName] string testName = null, Func<WriteContext, bool> expectedErrorsFilter = null)
+        public virtual IDisposable StartVerifiableLog(out ILoggerFactory loggerFactory, LogLevel minLogLevel, [CallerMemberName] string testName = null, Func<WriteContext, bool> expectedErrorsMatch = null)
         {
             var disposable = StartLog(out loggerFactory, minLogLevel, testName);
 
-            return new VerifyNoErrorsScope(loggerFactory, disposable, expectedErrorsFilter);
+            return new VerifyNoErrorsScope(loggerFactory, disposable, expectedErrorsMatch);
```

Comment:
shall we create another scope instead of `VerifyNoErrorsScope`?

Reply:
Currently this scope verifies both expected errors and no others errors. If separate the scope, it can verify one thing at a time. Is this expected?